instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java

```diff
@@ -143,6 +141,9 @@ public OpenTelemetry openTelemetry(
 
       ContextPropagators propagators = propagatorsProvider.getIfAvailable(ContextPropagators::noop);
 
+      // global is needed for logging appenders
+      GlobalOpenTelemetry.set(OpenTelemetrySdk.builder().setLoggerProvider(loggerProvider).build());
```

Comment:
this is my proposed compromise related to https://github.com/open-telemetry/opentelemetry-java/blob/main/docs/sdk-configuration.md#the-global-instance-of-the-sdk

Reply:
Hmm, this looks a bit hacky -- at the same time, I don't think I have any strong opinion on how we should solve this problem. Let's merge it as it is, and discuss the global issue on the SIG meeting